jest-preset.unit.js

```diff
@@ -4,7 +4,10 @@ module.exports = {
   setupFilesAfterEnv: [__dirname + '/test/unit.setup.js'],
   modulePathIgnorePatterns: ['.cache', 'dist'],
   testPathIgnorePatterns: ['.testdata.js', '.test.utils.js'],
-  testMatch: ['**/__tests__/**/*.[jt]s?(x)'],
+  testMatch: ['**/__tests__/**/*.{js,ts,jsx,tsx}'],
```

Comment:
Kept the jsx,tsx.. should not be necessary but in case a wrong test file is created, swc should fail.